sysopen/README.md

```diff
@@ -0,0 +1,5 @@
+# `package sysopen`
+
+The code in this folder is adapted from Bombadillo, you can see the original [here](https://tildegit.org/sloum/bombadillo/src/branch/master/http). Many thanks to Sloum and the rest of the team!
+
+The code is simple, and I have changed it, but in any case there should be no licensing issues because both repos are under GPL v3.
```

Comment:
I don't think there's any point in having this. If anything, it should say it was adapted from `webbrowser`, which is more accurate. But I think it can just be removed.